src/modules/Home.js

```diff
@@ -0,0 +1,213 @@
+    //clear tasks
+    const incomptasks = document.createElement("div");
+    incomptasks.classList = "position-relative container-fluid  ";
+    const clearButton = document.createElement("button");
+    clearButton.innerHTML = "Clear all completed";
+    clearButton.classList =
+      "  border p-3 container text-center text-secondary position-absolute ";
+    clearButton.style.left = "0";
+    clearButton.onclick = (e) => {
+      tasks = tasks.filter((el) => !el.complete);
+      tasks.map((el, i) => (el.index = i));
+      localStorage.setItem("tasks", JSON.stringify(tasks));
+      this.displayHome();
+    };
+    ///display incomplete tasks
+    const incompleteTasks = document.createElement("div");
+    incompleteTasks.innerHTML = "";
+
+    incompleteTasks.classList = " w-100 ";
+    incompleteTasks.style.height = (tasks.length * 60).toString() + "px";
+
+    tasks.map((task, i) => {
+      const incompleteTask = document.createElement("div");
+      incompleteTask.style.cursor = "move";
+      incompleteTask.style.left = "0";
+      incompleteTask.style.zIndex = "1";
+      incompleteTask.style.top = (task.index * 60).toString() + "px";
+
+      incompleteTask.style.height = "60px";
+      let diffInPositions = 0,
+        startingPosition = 0;
+      incompleteTask.ondblclick = (e) => {
+        incompleteTask.style.zIndex = "10";
+        e.preventDefault();
+        startingPosition = e.clientY;
+
+        incompleteTask.onmousemove = (e) => {
+          e.preventDefault();
+          diffInPositions = startingPosition - e.clientY;
+          startingPosition = e.clientY;
+
+          incompleteTask.style.top =
+            incompleteTask.offsetTop - diffInPositions + "px";
+          incompleteTask.style.left = "0";
+        };
+        incompleteTask.onmouseup = (e) => {
+          // if(incompleteTask.style.top < incompleteTasks.style.height)
+          e.preventDefault();
+          task.index =
+            Math.abs(incompleteTask.offsetTop) < Math.abs(container.offsetTop)
+              ? tasks.length - 1
+              : Math.floor(Math.abs(incompleteTask.offsetTop / 60));
+          console.log(task.index);
+          tasks.splice(i, 1);
+          tasks.splice(task.index, 0, task);
+          tasks.map((el, i) => (el.index = i));
+          localStorage.setItem("tasks", JSON.stringify(tasks));
+
+          this.displayHome();
+        };
+      };
+
+      incompleteTask.classList =
+        " d-flex border p-3 container justify-content-between align-items-baseline bg-white position-absolute  ";
+      const taskContainerDesc = document.createElement("div");
+      taskContainerDesc.classList =
+        "d-flex align-items-baseline justify-content-between ";
+      const incompleteTaskCheck = document.createElement("input");
+      incompleteTaskCheck.type = "checkbox";
+      incompleteTaskCheck.onclick = (e) => {
+        e.preventDefault();
+        if (task.complete === true) {
+          //true
```

Comment:
- It is good to use conditional shorthands. you don't need to check it `if(task.complete)` is enough.

Reply:
Thanks for pointing this out, I believe it is fixed now/